new-frontiers-14/frontier-station-14
# Add NFSD medals, The Glorious Order

## About the PR



Adds an NFSD-specific medal case and a set of seven new medals for different feats (both in service and for civilians).  Adds a craftable gaudy monstrosity of a medal for pompous idiocy.

Thank you to GhostPrince for the sprites, I hope my minor touchups (the lengths of the ribbon on the bronze and silver star, striping on the gold star) and pallet swaps (civilian medal ribbons) aren't awful.

## Why / Balance


Recognition of effort is good and promotes roleplay.  The Glorious Order is a dumb joke item, but I think it's funny.

## How to test


1. Warp to/spawn on NFSD Outpost
2. Enter the Sheriff's office.
3. Look at the medal case, admire the little golden shield.
4. Open the medal case, inspect the names, descriptions, and images of the medals.  Try them on, they should all only show when your character is facing the camera.
5. Craft The Glorious Order, it should give you enough time to contemplate ruining several perfectly good medals for a stupid gimmick.
6. Try it on, it should show when the character is facing the camera, and on both side views.

## Media


![image](https://github.com/user-attachments/assets/0136d2d3-1b9f-43eb-8d42-c8a4c75d5a1f)


## Breaking changes


**Changelog**



:cl:
- add: The NFSD has a new, more appropriate set of medals to be given out.
- add: A new medal, the Glorious Order of the Frontier, 1st Class can now be crafted.
